packages/modular-scripts/src/utils/gitActions.ts

```diff
@@ -55,10 +55,9 @@ function getModifiedAndUntrackedFileChanges(): string[] {
 }
 
 // Get all diffed files from git remote origin HEAD
-function getGitDiff(): string[] {
-  const defaultBranch = getGitDefaultBranch();
+function getGitDiff(targetBranch: string): string[] {
```

Comment:
Is this def a backwards compatible change btw? I can see that the lint task relies on it

Reply:
This function is not exported - it's only internal and used in `getDiffedFiles`, which now has a default argument (so the lint task can invoke it as it did before)